code-challenges.js

```diff
@@ -59,7 +87,10 @@ const givenValue2 = 10
 
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: .sort method is used to compare values.
+
+console.log(typeof dataTypes.length) and then add the .sort method
+console.log(sanDiegoSummerTemperature.sort)
```

Comment:
This is so close! The .sort((a, b) => b - a) is the correct way for the console.log to sort them from largest to smallest.